integration-tests/src/test/resources/features/ingress.feature

```diff
@@ -38,7 +38,7 @@ Feature: Ingress tests
     Then the Ingress of the Bridge "mybridge" metric 'http_server_requests_seconds_count{method="POST",outcome="SUCCESS",status="200",uri="/events",}' count is at least 1
 
   Scenario: Send plain Cloud Event
-    When send a cloud event to the Ingress of the Bridge "mybridge" with path "events/plain" and default headers:
+    When send a cloud event to the Ingress of the Bridge "mybridge" with path "events/plain" and headers "ce-id":"my-id","ce-source":"mySource","ce-specversion":"1.0","ce-type":"myType":
```

Comment:
do we want to reuse the `my-id` into the JSON events ?

Reply:
My idea is to leverage the event id in the steps checking connected systems, something like to check that the Slack channel contains the message with text `hello <cloud-event.my-id.id>`

Reply:
Well, I used `my-id` here because this is the value which was used internally in original implementation and I didn't bother to think up anything else. I didn't update the id for other tests, as currently the value is not used in following steps in scenario so I didn't need to put there anything simple and easily recognizable.

Reply:
I didn't update other tests as the change won't have any effect there, would just inflate the PR size.

Reply:
fair enough :)